res/layout/conversation_item_update.xml

```diff
@@ -8,8 +8,8 @@
     android:focusable="true"
     android:gravity="center"
     android:orientation="vertical"
-    android:paddingTop="13dp"
-    android:paddingBottom="13dp"
+    android:paddingTop="5dp"
+    android:paddingBottom="7dp"
```

Comment:
having the same amount atop and abottom leads irregular paddings; i do not want to refactor layout in this pr, therefore just so :)